src/main.js

```diff
@@ -100,6 +85,28 @@ utils.logger = winston.createLogger({
   ],
 });
 
+// config must be loaded after the logger is initialized
+configManager.loadConfig();
+
+if (configManager.config.discordLogging.active ?? true) {
```

Comment:
When config is completely empty, error. When the `discordLogging` key is defined, all is good.
```
[2022-04-24 18:28:08] [error] uncaughtException: Cannot read properties of undefined (reading 'active')
TypeError: Cannot read properties of undefined (reading 'active')
    at Object.<anonymous> (/home/brandon/Documents/git/devcodeabode/MoDiBo/src/main.js:91:41)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
    ```